XIV on Mac/en.lproj/Localizable.strings

```diff
@@ -154,9 +155,20 @@ Do not enable this on non-retina displays, unless for screenshot purposes.";
 "SETTINGS_PLATFORM_STEAM" = "Steam";
 "SETTINGS_FREE_TRIAL" = "Free Trial";
 "SETTINGS_GENERAL_FREE_TRIAL_BLURB" = "When using the Free Trial, you must choose the same license type you used when you signed up for the trial. Additionally, your Final Fantasy XIV account must already have an active trial, as XIV on Mac cannot sign you up automatically.";
+"SETTINGS_GENERAL_TITLE_NETWORK" = "Network";
 "SETTINGS_DOWNLOAD_LIMIT" = "Limit Download speed to";
 "SETTINGS_DOWNLOAD_LIMIT_PLACEHOLDER" = "0.0";
 "SETTINGS_DOWNLOAD_LIMIT_UNITS" = "MB/sec";
+"SETTINGS_DOWNLOAD_LIMIT_UNITS" = "MB/sec";
```

Comment:
These line seems to be double now; on a side note can you copy paste the newly added strings to the other languages as well until they are localized?